src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((pair) => pair.split(':').map((str) => str.trim()))
+    .filter(([property, value]) => property && value)
+    .reduce((styles, [property, value]) => {
+      styles[property] = value;
+
+      return styles;
+    }, {});
```

Comment:
The function implementation is correct and follows the best practices outlined in the checklist. However, there is a small concern that the '===' operator in the usage example will not work as expected because objects in JavaScript are compared by reference, not by value. To compare the contents of two objects, you would need to use a deep equality check function.